src/shared/modules/settings/settingsDuck.js

```diff
@@ -37,6 +37,7 @@ export const getBrowserSyncConfig = (state) => {
 export const getMaxNeighbours = (state) => state[NAME].maxNeighbours || initialState.maxNeighbours
 export const getMaxRows = (state) => state[NAME].maxRows || initialState.maxRows
 export const getInitialNodeDisplay = (state) => state[NAME].initialNodeDisplay || initialState.initialNodeDisplay
+export const getScrollToTop = (state) => state[NAME].scrollToTop || initialState.scrollToTop
```

Comment:
@pe4cey I believe default value should be true to preserve previous behaviour. Because the setting is new and so gets false by default